xbmc/input/InputManager.h

```diff
@@ -289,6 +294,12 @@ class CInputManager : public ISettingCallback,
   std::vector<KODI::MOUSE::IMouseDriverHandler*> m_mouseHandlers;
 
   std::unique_ptr<KODI::KEYBOARD::IKeyboardDriverHandler> m_keyboardEasterEgg;
+
+  // Input state
+  bool m_enableController = true;
+
+  // Settings
+  static const std::string SETTING_INPUT_ENABLE_CONTROLLER;
```

Comment:
Sorry to bother again. Does this have to be a member if it's only used in this class? I find defining these in an anonymous namespace to be nice and tidy. It's totally up to you though as I'm fine either way :)